sda-commons-server-openapi/src/test/java/org/sdase/commons/server/openapi/apps/test/Filter.java

```diff
@@ -0,0 +1,15 @@
+package org.sdase.commons.server.openapi.apps.test;
+
+public class Filter {
```

Comment:
I wonder if we should extend the example to a more common use case because we usually support multiple filter types. In that case `Filter` is abstract and defines `@JsonSubTypes`. Your class would be a specific `AnimalFilter`. Any opinion, @JoergSiebahn ?

Reply:
As I understand the implementation and the changes (it's actually very little understanding), that detail should not affect the case that is solved in the changed modifier. I think that introducing inheritance here is just more overhead.